### PR TITLE
Authority list and group resource fix

### PR DESCRIPTION
**Pull request by beckyconning**

### CODE REVIEW

**Comment by beckyconning** [comment]
@cryogenian please review

**Comment by cryogenian** [comment]
Could you please add `parseRoot` to other group related things?

**Comment by beckyconning** [comment]
Could you point them out to me? I thought it was only the `decodeJSON` instance that needed changing.

**Comment by cryogenian** [comment]
`GrantedTo`, `ShareableSubject`, `ShareRequest` and `EncodeJson` instances too

File: src/Quasar/Advanced/Types.purs [hunk 1]
```diff
@@ -63,19 +63,24 @@ instance decodeJsonAccessType ∷ DecodeJson AccessType where
...
+  | Group (Either Root (Pt.AbsFile Pt.Sandboxed))
 
 derive instance eqResource ∷ Eq Resource
 derive instance ordResource ∷ Ord Resource
 
 instance encodeJsonResource ∷ EncodeJson Resource where
   encodeJson (File pt) = encodeJson $ "data:" <> Pt.printPath pt
   encodeJson (Dir pt) = encodeJson $ "data:" <> Pt.printPath pt
-  encodeJson (Group pt) = encodeJson $ "group:" <> Pt.printPath pt
+  encodeJson (Group (Right pt)) = encodeJson $ "group:" <> Pt.printPath pt
```
Comment (beckyconning) [inline src/Quasar/Advanced/Types.purs]: is this not the change needed to the `EncodeJson` instance? sorry if i'm missing something : ).

Comment (cryogenian) [reply src/Quasar/Advanced/Types.purs]: Indeed :smile: :+1:

**Pull request merged**